src/pages/index.js

```diff
@@ -78,7 +78,7 @@ const Index = ({ data }) => {
 };
 Index.propTypes = {
   data: PropTypes.object.isRequired,
-  search: PropTypes.object.isRequired,
+  search: PropTypes.object,
```

Comment:
Why are we doing this? If it isn't necessary let's remove it. Otherwise, let's establish a reason.